### DISCUSSION TITLE
The ‘Home’ value of Breadcrumb is recognized as empty by Google.

### DISCUSSION THREAD

**Discussion by stan1781** [bootscore/bootscore#775] [Q&A] [answered]
The default ‘Home’ for Breadcrumbs is set as an icon, which causes Google search to not recognize the website’s schema. The error message is as follows: 
‘Either “name” or “item.name” should be specified (in “itemListElement”). Items with this issue are invalid. Invalid items are not eligible for Google Search’s rich results.’ 

Can the icon for ‘home’ be changed to ‘Home’?”

To address the issue, you can indeed change the icon to a text label ‘Home’ in your schema markup. This will provide a ‘name’ value for the home item, which should resolve the error and make your items eligible for Google’s rich results.

**Comment by crftwrk** [comment, marked as answer]
Of course, you can edit the breadcrumb https://bootscore.me/documentation/functions/#Example_Changing_the_breadcrumb

But in general it might be a good idea to add a `visually-hidden` Home text to the icon to fix that. Are you able to create a PR?

**Comment by stan1781** [comment]
I’m sorry, I would like to submit a ‘PR’ as well, but I’m just a hobbyist programmer and don’t know how to submit a ‘PR’.

**Reply by crftwrk** [reply]
>  I’m just a hobbyist programmer

You provided already some nice workarounds and found serious issues. Creating a PR is just a few clicks:

- Go to the [theme page](https://github.com/bootscore/bootscore) and press the 'Fork' button at the top.
- Now you own a copy of the theme and you can edit it as you want. Go in your copy to:

https://github.com/bootscore/bootscore/blob/8153fe6bdf6d7dd2204728154296ebf3ac77372e/inc/breadcrumb.php#L24

and change this line into:

```php
 echo '<li class="breadcrumb-item"><a href="' . home_url() . '">' . '<i class="fa-solid fa-house"></i><span class="visually-hidden">' . __('Home', 'bootscore') . '</span>' . '</a></li>';
```

- After changes have been made, GitHub will ask you if you want to contribute and create a Pull Request (PR).
- Press yes, that's it

You will see that this is easy, feels good and makes fun.

**Reply by stan1781** [reply]
This modification is good, but there are still bugs in the WooCommerce product categories and product pages. I modified the code in woocommerce/inc/wc-breadcrumb.php, but it seems that it cannot output the <span> tag.
`  // Remove "Home" and add Fontawesome house icon (_wc_breadcrumb.scss)
        'home'        => '<span class="visually-hidden">' . _x('Home', 'breadcrumb', 'woocommerce') . '</span>',
      //'home'        => ' ',
`

**Reply by crftwrk** [reply]
Yes, WooCommerce breadcrumb is different and Woo escapes the HTML. That's why the icon is solved the CSS way here. Think we should fix the default breadcrumb first and find a solution for Woo breadcrumb in a second step. Maybe ChatGDP has an idea.

**Reply by crftwrk** [reply]
I see that you have created https://github.com/bootscore/bootscore/compare/main...stan1781:bootscore:main. Do you want to simply press the green "Create pull request" button? Then we have this fixed and can search for a solution for WC breadcrumb.

**Reply by crftwrk** [reply]
It seems that the WC breadcrumb is harder to handle than expected. I've added a note to v.6.1 to fix it there.

**Reply by crftwrk** [reply]
https://github.com/bootscore/bootscore/pull/821
